Marlin/Marlin_main.cpp

```diff
@@ -457,7 +457,7 @@ static int serial_count = 0;
 // Inactivity shutdown
 millis_t previous_cmd_ms = 0;
 static millis_t max_inactive_time = 0;
-static millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
+millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
 
```

Comment:
Then why does a RC8-RCBugFix install, with minimal appropriate changes for coreXY ignore an M84 S0, and deactivate steppers after the time indicated in configuration_adv.h?  That is exactly the issue I'm running in to. 

After reading marlin_main.cpp , the "static millis_t stepper_inactive_time" is set at initial run. And this declaration is a global, being outside functions declared in marlin_main.cpp .

So, when a M84 S## proceeds to change it, you're in marlin_main.cpp::gcode_M18_M84() , which is IN SCOPE of the set static variable and therefore cannot be changed. We don't ever actually exit marlin_main.cpp during execution , so this variable is never unset. It points to 1 memory location, and that's the initial set in global scope on line 460.

That's why the constant set in configuration_adv.h::#define DEFAULT_STEPPER_DEACTIVE_TIME 120 is what's used no matter what.